bin/run.sh

```diff
@@ -24,16 +24,22 @@ test_file="${2}/${1}.spec.js"
 result_file="${3}/results.json"
 
 # Change xtest to test so all tests are run
-sed -i 's/xtest/test/g' "${test_file}"
-sed -i 's/xit/it/g' "${test_file}"
+if [[ "$OSTYPE" == "darwin"* ]]; then # Mac OS X
+  # BSD sed -i takes an extra parameter to specify the backup file extension
+  sed -i 'tmp' 's/xtest/test/g' "${test_file}"
+  sed -i 'tmp' 's/xit/it/g' "${test_file}"
+else # linux
```

Comment:
I think this will work just fine on WSL / windows as well. 

```suggestion
else
```